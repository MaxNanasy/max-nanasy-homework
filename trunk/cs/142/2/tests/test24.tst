main(){	while(i!=a[0]){		a[0] = a[1] + a[i[0]];	}}