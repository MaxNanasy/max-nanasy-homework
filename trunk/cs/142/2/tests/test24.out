program
   statementSequence
      statement
         whileStatement
            condition
               expression
                  term
                     factor
                        designator
               expression
                  term
                     factor
                        designator
                           expression
                              term
                                 factor
            statementSequence
               statement
                  assignment
                     designator
                        expression
                           term
                              factor
                     expression
                        term
                           factor
                              designator
                                 expression
                                    term
                                       factor
                        term
                           factor
                              designator
                                 expression
                                    term
                                       factor
                                          designator
                                             expression
                                                term
                                                   factor
